lbrynet/reflector/util.py

```diff
@@ -0,0 +1,47 @@
+import logging
```

Comment:
in general, avoid using general names like `util` for things that can otherwise be named.  this could be called, for example, `reupload.py`